browser-test-helper.js

```diff
@@ -1,7 +1,8 @@
 // eslint-disable-next-line no-var
-var pkg = require('./package.json');
+window.pkg = require('./package.json');
 
-window.pkg = pkg;
-window.STREAM_API_KEY = process.env.STREAM_API_KEY;
-window.STREAM_ANALYTICS_TOKEN = process.env.STREAM_ANALYTICS_TOKEN;
-window.STREAM_BASE_URL = process.env.STREAM_BASE_URL;
+window.env = {
+    STREAM_API_KEY: process.env.STREAM_API_KEY,
+    STREAM_ANALYTICS_TOKEN: process.env.STREAM_ANALYTICS_TOKEN,
+    STREAM_BASE_URL: process.env.STREAM_BASE_URL,
+};
```

Comment:
:+1:

Reply:
but maybe with `stream_` prefix, window is global at the end. We don't want to mess others' env.

Reply:
and then we don't need `STREAM_` per property.

Reply:
fixed, could you update the github env vars?

Reply:
himm, I think we should keep `STREAM_` prefix in environment variables but good to remove inside set env. Something like:

```js
window.__stream_env__ = {
  API_KEY: process.env.STREAM_API_KEY,
  ...
}
```